routing_model/index.html

```diff
@@ -289,47 +293,108 @@ <h2>Deciding when to accept an incoming relocation</h2>
       VoteParsecExpectCandidate --> LoopEnd
 
       Balanced(("Check?<br/>(shared state)"))
       Balanced -- "shorter_prefix_section(<br/>).is_some()" --> Resend
-      Balanced -- "shorter_prefix_section().is_none()<br/>PROCESSING_CANDIDATE==yes" --> SendRefuse
-      Balanced -- "shorter_prefix_section().is_none()<br/>PROCESSING_CANDIDATE==No" --> SetCandidateYes
+      Balanced -- "shorter_prefix_section().is_none()<br/>!waiting_proofing().is_empty()" --> SendRefuse
+      Balanced -- "shorter_prefix_section().is_none()<br/>waiting_proofing().is_empty()" --> SendRelocateResponse
 
       Resend["send_rpc(<br/>ExpectCandidate)<br/>to shorter_prefix_section()"]
       Resend --> LoopEnd
 
       SendRefuse["send_rpc(<br/>RefuseCandidate)<br/>to source section"]
       SendRefuse --> LoopEnd
 
-      SetCandidateYes["PROCESSING_CANDIDATE=yes<br/>(shared state)"]
-      SetCandidateYes-->Concurrent0
+      SendRelocateResponse["add_node(NodeState=WaitingCandidateInfo)<br/><br/>send_rpc(RelocateResponse)<br/>to source section"]
+      SendRelocateResponse --> LoopEnd
 
-      Concurrent0{"Concurrent<br/>paths"}
-      Concurrent0 --> AcceptAsCandidate
-      Concurrent0 --> LoopEnd
+  </div>
+
+  <h2>Relocated node connection</h2>
+  <div class=description>
+  <p>Manage node with NodeState=WaitingConnectionInfo and connection info request/response.<br/>
+    CandidateInfo is sent to NaeManager of the target interval<br/>
+    When we complete, we either stop responding to CandidateInfo RPCs, or we send NodeConnected RPC<br/>
+  </p>
+  </div>
+  <div class="mermaid">
+      graph TB
+      RelocatedNodeConnection["Start Relocated Node Connection"]
+      style RelocatedNodeConnection fill:#f9f,stroke:#333,stroke-width:4px
+
+      RelocatedNodeConnection --> StartCheckRelocatedNodeConnectionTimeout
+      StartCheckRelocatedNodeConnectionTimeout["schedule(<br/>CheckRelocatedNodeConnectionTimeout)"]
+      StartCheckRelocatedNodeConnectionTimeout --> LoopStart
+
+      WaitFor(("Wait for 2:"))
+
+      LoopStart-->WaitFor
+
+      WaitFor -- Consensus--> ParsecConsensus
+      ParsecConsensus((Consensus))
+
+      ParsecConsensus -- "ParsecCheckRelocatedNodeConnection" --> CleanCandidates
+      CleanCandidates["for candidate in both<br/>-waiting_node_connecting()<br/>-CANDIDATES:<br/><br/>purge_node_info(candidate)"]
+      CleanCandidates --> ReStartCheckRelocatedNodeConnectionTimeout
+      ReStartCheckRelocatedNodeConnectionTimeout["CANDIDATES=waiting_node_connecting()<br/><br/>schedule(<br/>CheckRelocatedNodeConnectionTimeout)"]
+      ReStartCheckRelocatedNodeConnectionTimeout --> LoopEnd
 
 
-      AcceptAsCandidate["AcceptAsCandidate<br/>(shared state)"]
-      style AcceptAsCandidate fill:#f9f,stroke:#333,stroke-width:4px
-      AcceptAsCandidate --> SetCandidateNo
+      ParsecConsensus -- "ParsecCandidateInfo<br/>and<br/>is_valid_waited_info(<br/>CandidateInfo)" --> SetCandidateInfo
```

Comment:
I think this would be clearer with a "check?" box in between `ParsecConsensus` and the 3 variations of `CandidateInfo`. Currently, it looks visually like there were 4 different RPCs to handle...